src/datastore_mad/remotes/downloader.sh

```diff
@@ -408,9 +431,17 @@ lxd://*)
     file_type="application/octet-stream"
     command="$VAR_LOCATION/remotes/datastore/lxd_downloader.sh \"$FROM\""
     ;;
-docker://*|dockerfile://*)
-    file_type="application/octet-stream"
-    command="$VAR_LOCATION/remotes/datastore/docker_downloader.sh \"$FROM\""
+restic://*|restic+rbd://*)
+    defs=`$VAR_LOCATION/remotes/datastore/restic_downloader.rb "$FROM" | grep -e '^command=' -e '^clean_command='`
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use $(...) notation instead of legacy backticks `...`.